src/common/Debug.ts

```diff
@@ -16,6 +16,11 @@
 
 import { isNode } from '../environment.js';
 
+declare global {
+  // eslint-disable-next-line no-var
+  var __PUPPETEER_DEBUG: string;
```

Comment:
can it be `const`? (not sure what TS supports here)

Reply:
It must be `var` because it is in the global scope.